sbc_translation/models/correspondence.py

```diff
@@ -456,6 +456,22 @@ def get_translated_elements(self):
                 })
         return res
 
+    @api.multi
+    def increment_priority_cron(self):
+        """
+        Increment priority of letters to translate, maximum
+        priority is 4.
+        """
+        letters_to_translate = self.search([("translation_status", "!=", "done")])
```

Comment:
Also filter the letters that have no translation_status:
```suggestion
        letters_to_translate = self.search([("translation_status", "not in", [False, "done"])])
```